toffy/test_utils.py

```diff
@@ -192,6 +196,29 @@ def check_qc_dir_structure(out_dir: str, point_names: List[str], qc_plots: bool
                 assert(os.path.exists(os.path.join(out_dir, '%s_barplot_stats.png' % mn)))
 
 
+def check_mph_dir_structure(plot_dir: str, point_names: List[str], combined: bool = False):
+    """Checks MPH directory for minimum expected structure
+
+    Args:
+        plot_dir (str):
+            Folder containing MPH plot output
+        point_names (list):
+            List of expected point names
+        combined (bool):
+            whether to check for combined mph data csv and plot image
+
+    Raises:
+        AssertionError:
+            Assertion error on missing csv
+    """
+    for point in point_names:
+        assert(os.path.exists(os.path.join(plot_dir, f'{point}-pulse_height.csv')))
```

Comment:
This should fail now that the name for this file changed. Make sure it is failing locally, and if not figure out why not